workers/unity/Assets/Playground/Scripts/MonoBehaviours/ToggleRotationCommandSender.cs

```diff
@@ -10,11 +10,22 @@ public class ToggleRotationCommandSender : MonoBehaviour
     {
         [Require] private SpinnerRotation.Requirables.Reader reader;
         [Require] private SpinnerRotation.Requirables.CommandRequestSender requestSender;
+        [Require] private SpinnerRotation.Requirables.CommandResponseHandler responseHandler;
         private EntityId ownEntityId;
 
         private void OnEnable()
         {
             ownEntityId = GetComponent<SpatialOSComponent>().SpatialEntityId;
+            responseHandler.OnSpinnerToggleRotationResponse += ResponseHandlerOnOnSpinnerToggleRotationResponse;
```

Comment:
Let's also deregister the callbacks in OnDisable() for now just to be safe and keep things consistent. :)

Reply:
aye